<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Abhishek" failures="3" tests="4" name="Banking App Test(failed)(failed)" time="36.586" errors="0" timestamp="2022-02-27T21:55:09 IST">
  <testcase classname="com.eBankingAutomationV1.testCases.TC_LoginTest_001" name="loginTest" time="2.872"/>
  <testcase classname="com.eBankingAutomationV1.testCases.TC_LoginDDT_001" name="loginDDT" time="6.942">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.eBankingAutomationV1.testCases.TC_LoginDDT_001.loginDDT(TC_LoginDDT_001.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.eBankingAutomationV1.testCases.TC_LoginDDT_001" name="loginDDT" time="6.805">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.eBankingAutomationV1.testCases.TC_LoginDDT_001.loginDDT(TC_LoginDDT_001.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.eBankingAutomationV1.testCases.TC_LoginDDT_001" name="loginDDT" time="6.1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.eBankingAutomationV1.testCases.TC_LoginDDT_001.loginDDT(TC_LoginDDT_001.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- Banking App Test(failed)(failed) -->
